src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java

```diff
@@ -211,13 +211,11 @@ public AddToCartResponse addSamplesToCart(@RequestBody AddToCartRequest addToCar
 	 *            The {@link Project} ID
 	 * @param sampleIds
 	 *            The {@link Sample} ID
-	 * @return a map stating success
 	 */
 	@RequestMapping(value = "/project/{projectId}/samples", method = RequestMethod.DELETE, consumes = MediaType.APPLICATION_JSON_VALUE)
 	@ResponseBody
```

Comment:
Does `@ResponseBody` do anything when the return type is `void`?

Reply:
Looks like you don't.  Removed in 4fc8d442ef68e3a6d778777f9c44454ef20b8210